dsl_parser/interfaces/utils.py

```diff
@@ -19,9 +19,11 @@
 from dsl_parser.exceptions import DSLParsingLogicException
 
 
-def validate_missing_inputs(inputs):
-    for key, value in inputs.iteritems():
-        if value is None:
+def validate_missing_inputs(inputs, inputs_schema):
+    """Check that all inputs defined in inputs_schema exist in inputs"""
+
```

Comment:
It is better to display all the missing inputs together, as it will prevent the user from getting this error multiple times in case of multiple missing inputs.
you can use something in the likes of:
`
missing_inputs = set(inputs_schema.keys() - inputs.keys())
if missing_inputs:
    ...
`